src/spatial/attitude-utils.hpp

```diff
@@ -185,9 +227,13 @@ class Quaternion {
     bool IsUnit(decimal tolerance) const;
     Quaternion Canonicalize() const;
 
+    /// The real component
     decimal real;
+    /// The i component
```

Comment:
nit: slightly redundant comment, variable name is already very clear

Reply:
For doxygen